src/sst/core/baseComponent.h

```diff
@@ -115,6 +115,8 @@ class BaseComponent
     int         getCurrentPriority() const;
     /** Return the elapsed simulation time as a time */
     UnitAlgebra getElapsedSimTime() const;
+    /** Return the end simulation time as a cycle count*/
+    SimTime_t   getEndSimCycle() const;
```

Comment:
We've got `getEndSimCycle()` and `getFinalSimTime()`. Can we change these to use the same term while we're at it? Changing getEndSimCycle to getFinalSimCycle is easiest since it's a new function anyways, but I do like "End" better.

Reply:
We can change them.  We can do either getFinalSimCycle() and gerFinalSimTime() or we can do getEndSimCycle() and getEndSimTime().  In the case of the second, I would just add both and deprecate getFinalSimCycle().  I don't have a strong preference, but I think "End" is somewhat better as well.